datachannel.go

```diff
@@ -40,6 +40,8 @@ type DataChannel struct {
 	readyState                 atomic.Value // DataChannelState
 	bufferedAmountLowThreshold uint64
 	detachCalled               bool
+	readLoopActive             chan struct{}
+	isUserClosed               bool
```

Comment:
`isGracefulClose`? I see this set via `Close` or `GracefulClose` only

Reply:
good idea, more specific :)